src/views/Activity.vue

```diff
@@ -345,63 +346,58 @@ export default {
       );
     },
 
-    windowTimelineQuery: function(windowbucket, afkbucket, host, starttime, endtime){
-      return 'NAME="window_timeline@'+host+'" \n\
-STARTTIME="'+starttime+'" \n\
-ENDTIME="'+endtime+'" \n\
-not_afk=query_bucket("'+afkbucket+'") \n\
-events=query_bucket("'+windowbucket+'") \n\
-not_afk=filter_keyval(not_afk, "status", "not-afk", FALSE) \n\
-events=filter_period_intersect(events, not_afk) \n\
-events=sort_by_timestamp(events) \n\
-RETURN=events';
+    windowTimelineQuery: function(windowbucket, afkbucket){
+      return { "query": [
+        'not_afk = query_bucket("'+afkbucket+'");',
```

Comment:
All variables used in the query need to be sanitized, we should at the very least add a TODO/FIXME comment.

Reply:
Done